Example/Tests/CDTQQuerySqlTranslatorTests.m

```diff
@@ -652,6 +652,21 @@
                 expect(parts.sqlWithPlaceholders).to.equal(@"(\"name\" IS NULL)");
             });
         });
+        
+        describe(@"when using the $in operator", ^{
+            it(@"constructs valid WHERE clause when using a single element in an array", ^{
+                CDTQSqlParts *parts = [CDTQQuerySqlTranslator
+                    wherePartsForAndClause:@[@{ @"name" : @{ @"$in" : @[ @"mike" ]} }]
+                                usingIndex:@"named"];
+                expect(parts.sqlWithPlaceholders).to.equal(@"\"name\" IN ( ? )");
+            });
+            it(@"constructs valid WHERE clause when using multiple elements in an array", ^{
+                CDTQSqlParts *parts = [CDTQQuerySqlTranslator
+                    wherePartsForAndClause:@[@{ @"name" : @{ @"$in" : @[ @"mike", @"fred" ]} }]
+                                usingIndex:@"named"];
+                expect(parts.sqlWithPlaceholders).to.equal(@"\"name\" IN ( ?, ? )");
+            });
```

Comment:
That is correct.  I believe that I copied one of the `$gt, $gte, $lt, $lte` tests, which also don't check the placeholders, as a template for my tests.  I updated all of them to check placeholders now.  Done in b667c92d0cf9994ca7a8751af6e7efedcb6f3199.